src/utility.js

```diff
@@ -92,7 +92,18 @@ function sum(x) {
   return x.reduce((a, b) => a + b, 0);
 }
 
-function mergeInto(obj, other) {
+// if allowOverride is false, it shows error in case of symbol redefinition
+function mergeInto(obj, other, allowOverride = true) {
```

Comment:
How about defining an "enum" for this, like so?

```js
mergeInfo(LibraryManager.library, { .. }, LibraryManager.DisallowOverride);
```
That would make it easier to read.

That name is just an idea. Could also be without the namespace, or a different name.

Reply:
Added LibraryParams